templates/base.html

```diff
@@ -68,13 +69,32 @@
   {% block content %}{% endblock content %}
 </div>
 
-
-<footer class="full-width-bg-container" id="footer-bg">
-  <div class="main-content-container" id="footer">
-    <p>SolvedChinese.org&#169;2019</p>
-    <a href="https://github.com/chenyx512/Jiezi" target="_blank"><img src="/static/images/github.svg" class="footer-icon"></a>
-  </div>
-</footer>
+<div class="footer" style="background: #ccc">
+        <a href="{% url 'index' %}">
+            <img src="{% static "images/base/logo.png" %}" style=" margin-left: 12vw; padding-left:3vw; height:4vw; vertical-align:middle">
```

Comment:
A couple of things:
- We are using the new logo now, it's [this file](https://github.com/chenyx512/jiezi/blob/master/static/images/logo-color-full.svg), please use the new logo.
- Indent HTML files with 2 spaces, and align the tags.
- Generally, a repeated style (e.g. `vertical-align: middle`) can be changed into a class, this is to reduce code redundancy and increase readability.
- For the mail icon, use font-awesome (we have Pro), not an image.